feat(Select): added errorMessage, errorPlacement and validationType props

kkirik opened the pull request:
Closes #1273

korvin89 commented on src/components/Select/components/SelectControl/SelectControl.tsx:
```diff
@@ -1,10 +1,12 @@
-import {ChevronDown} from '@gravity-ui/icons';
+import {ChevronDown, TriangleExclamation} from '@gravity-ui/icons';
 import isEmpty from 'lodash/isEmpty';
+import type {CnMods} from 'src/components/utils/cn';
```
We don't support aliases. Btw, CI tests failed because of it

kkirik replied:
Thanks. I'll check

korvin89 commented:
One more thing: could u add some tests for this features? U can get tests from [here](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L91). And add QA constant [here](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/constants.ts#L34) please. And there is some different behaviour regarding TextInput (in [this](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L98) test case, but it has a confusing name :))

kkirik commented:
> One more thing: could u add some tests for this features? U can get tests from [here](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L91). And add QA constant [here](https://github.com/gravity-ui/uikit/blob/main/src/components/Select/constants.ts#L34) please. And there is some different behaviour regarding TextInput (in [this](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/__tests__/TextInput.input.test.tsx#L98) test case, but it has a confusing name :))

So, I added some tests for error feature also I added an aria-label attribute for getting internal error icon by i11y attributes.
